operator/cmd/mesh/testdata/manifest-generate/output/all_on.golden-show-in-gh-pull-request.yaml

```diff
@@ -1,723 +1,57 @@
-# Resources for AddonComponents coreDNS component
-
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: istiocoredns
+  name: istio-reader-istio-system
```

Comment:
something seems to have gone wrong there, the resources look to be reordered and so it's hard to compare. also, @ayj 
 did some magic to make this particular file expand by default in GH review and that seems to not by happening anymore.

Reply:
There was actually something wrong: CoreDNS was disabled by my changes. Fixed that now. The re-ordering is still there though

Reply:
And another issue: all manifests had been duplicated. Fixed now